godot-core/src/builtin/arrays.rs

```diff
@@ -495,6 +495,7 @@ impl_builtin_traits! {
         Default => array_construct_default;
         Clone => array_construct_copy;
         Drop => array_destroy;
+        Eq => array_operator_equal;
```

Comment:
As mentioned, this should probably be `PartialEq` until we clarify the behavior of `NaN` in Godot.